scripts/js/commands/api/updateApiDocs.ts

```diff
@@ -73,33 +73,19 @@ const readArgs = (): Arguments => {
     .parseSync();
 };
 
-zxMain(async () => {
-  const args = readArgs();
```

Comment:
No changes made to the logic or sequencing of this code block. I only repositioned it so that the new function `generateVersion()` abstracts some of it, and the main is at the bottom of the file.